baseplate/secrets/fetcher.py

```diff
@@ -318,8 +318,11 @@ def main():
         level = logging.DEBUG
     else:
         level = logging.INFO
-    logging.basicConfig(level=level)
 
+    logging.basicConfig(
```

Comment:
syslog handles this in normal prod, does k8s require this?

Reply:
as long as you're emitting to stdout/stderr, it'll be easily reachable (and shippable once we plug into Ricky's logging work)

Reply:
sorry, i meant adding a timestamp to the log line

Reply:
Whoops, misunderstood. It will not modify the log output. Once we have log shipping in place, the agent would be attaching that as structured metadata.